modules/nf-core/islandpath/tests/main.nf.test

```diff
@@ -0,0 +1,47 @@
+// nf-core modules test islandpath
+nextflow_process {
+
+    name "Test Process ISLANDPATH"
+    script "../main.nf"
+    process "ISLANDPATH"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gunzip"
+    tag "islandpath"
+
+    test("bacteroides_fragilis - gbff") {
+
+        setup {
+            run("GUNZIP") {
+                script "../../gunzip/main.nf"
+                process {
+                    """
+                    input[0] = Channel.fromList([
+                                    tuple([ id:'test' ], // meta map
+                                    file(params.test_data['bacteroides_fragilis']['genome']['genome_gbff_gz'], checkIfExists: true)),
```

Comment:
New way of writing file paths now, instead of this:
file(params.test_data['bacteroides_fragilis']['genome']['genome_gbff_gz'], checkIfExists: true)),
do this:
file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/genome/genome.gbff.gz', checkIfExists: true)

Reply:
Got it, will do.

Reply:
Done in [9e9a297](https://github.com/nf-core/modules/pull/5321/commits/9e9a29756df4ae1597e162a61ac52a2eb188646e)